sr_robot_lib/test/motor_updater_test.cpp

```diff
@@ -107,13 +108,17 @@ class MotorUpdaterTest
         }
       }
 
-
-      if (fabs(time_spent.toSec() - (static_cast<double>(static_cast<int>(time_spent.toSec())))) < tolerancy)
+      if (command->from_motor_data_type == MOTOR_DATA_CAN_NUM_RECEIVED)
       {
-        if (command->from_motor_data_type == MOTOR_DATA_CAN_NUM_RECEIVED)
+        can_num_transmitted_counter++;
+           if (fabs(time_spent.toSec() - (can_num_transmitted_counter * test3.when_to_update)) < tolerancy)
```

Comment:
```suggestion
        if (fabs(time_spent.toSec() - (can_num_transmitted_counter * test3.when_to_update)) < tolerancy)
```

Reply:
interestingly roslint cpplint did not find this offset. Sorry. Fixed